modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/VolatilePageMemoryStorageEngine.java

```diff
@@ -51,6 +54,13 @@ public class VolatilePageMemoryStorageEngine implements StorageEngine {
     /** Engine name. */
     public static final String ENGINE_NAME = "aimem";
 
+    /** 
+     * Maximum "work units" that are allowed to be used during {@link BplusTree} destruction.
+     *
+     * @see BplusTree#startGradualDestruction
+     */
+    public static final int MAX_DESTRUCTION_WORK_UNITS = 1_000_000;
```

Comment:
I would also make it `1_000` so that we can clean indexes from different regions quickly and clean memory more evenly.

Reply:
Fixed